modules/drivers/sqlserver/resources/metabase-plugin.yaml

```
@@ -21,6 +21,10 @@ driver:
       placeholder: N/A
     - user
     - password
+    - name: role
+      display-name: Impersonation User (required for impersonation if the default user cannot be impersonated)
+      helper-text: Specify a user to override the default user. Required if the default user cannot be impersonated, such as the 'sa' user.
```

Had to re-read this many times, is this better?

```suggestion
       display-name: Default Impersonation User (required for impersonation if the connection's user cannot be impersonated)
      helper-text: Specify a user to act as the default user. Required if the connection's user cannot be impersonated, such as the 'sa' user.
```